src/masternodes/rpc_icxorderbook.cpp

```diff
@@ -146,33 +178,37 @@ UniValue icxcreateorder(const JSONRPCRequest& request) {
     if (!metaObj["tokenFrom"].isNull()) {
         tokenFromSymbol = trim_ws(metaObj["tokenFrom"].getValStr());
 
-        if (!metaObj["chainTo"].isNull()) order.chain = trim_ws(metaObj["chainTo"].getValStr());
-        else throw JSONRPCError(RPC_INVALID_PARAMETER,"Invalid parameters, argument \"chainTo\" must be non-null if \"tokenFrom\" specified");
+        if (!metaObj["chainTo"].isNull())
+            order.chain = trim_ws(metaObj["chainTo"].getValStr());
+        else
+            throw JSONRPCError(RPC_INVALID_PARAMETER,"Invalid parameters, argument \"chainTo\" must be non-null if \"tokenFrom\" specified");
 
-        if (!metaObj["ownerAddress"].isNull()) order.ownerAddress = DecodeScript(metaObj["ownerAddress"].getValStr());
-        else throw JSONRPCError(RPC_INVALID_PARAMETER,"Invalid parameters, argument \"ownerAddress\" must be non-null if \"tokenFrom\" specified");
+        if (!metaObj["ownerAddress"].isNull())
+            order.ownerAddress = DecodeScript(metaObj["ownerAddress"].getValStr());
+        else
+            throw JSONRPCError(RPC_INVALID_PARAMETER,"Invalid parameters, argument \"ownerAddress\" must be non-null if \"tokenFrom\" specified");
        
-        if (!::IsMine(*pwallet, DecodeDestination(metaObj["ownerAddress"].getValStr()))) {
+        if (!::IsMine(*pwallet, DecodeDestination(metaObj["ownerAddress"].getValStr())))
```

Comment:
Use `order.ownerAddress` in other places as well, you don't need to decode it second time.